### PR TITLE
REF: reworded error msg

### PR DESCRIPTION
**Pull request by Overtorment**

### CODE REVIEW

File: models/signer.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ exports.createHDTransaction = function(utxos, toAddress, amount, fixedFee, chang
   }
 
   if (unspentAmountSatoshi < amountToOutputSatoshi + feeInSatoshis) {
-    throw new Error('Not enough confirmed inputs');
+    throw new Error('Not enough balance. Try sending smaller amount');
```
Comment (ncoelho) [inline models/signer.js]: Text can be "Not enough balance. Try to send a smaller amount"

File: models/signer.js [hunk 2]
```diff
@@ -88,7 +88,7 @@ exports.createHDSegwitTransaction = function(utxos, toAddress, amount, fixedFee,
...
```
Comment (ncoelho) [inline models/signer.js]: Code is replicated in two places?

Comment (Overtorment) [reply models/signer.js]: yep. its signer model for 2 different wallet types

File: models/signer.js (hunk 1 above)
Comment (Overtorment) [reply models/signer.js]: can be or should be?

Comment (Overtorment) [reply models/signer.js]: @ncoelho

Comment (ncoelho) [reply models/signer.js]: should be

**Pull request merged**